docs/userguide/networking/dockernetworks.md

```diff
@@ -0,0 +1,368 @@
+          TX packets:8 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:648 (648.0 B)  TX bytes:648 (648.0 B)
+
+lo        Link encap:Local Loopback
+          inet addr:127.0.0.1  Mask:255.0.0.0
+          inet6 addr: ::1/128 Scope:Host
+          UP LOOPBACK RUNNING  MTU:65536  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+
+/ # ping container1 -w3
+PING container1 (172.17.0.2): 56 data bytes
+64 bytes from 172.17.0.2: seq=0 ttl=64 time=0.277 ms
+64 bytes from 172.17.0.2: seq=1 ttl=64 time=0.179 ms
+64 bytes from 172.17.0.2: seq=2 ttl=64 time=0.130 ms
+--- container1 ping statistics ---
+3 packets transmitted, 3 packets received, 0% packet loss
+round-trip min/avg/max = 0.113/0.174/0.277 ms
+/ # cat /etc/hosts
+172.17.0.3      bda12f892278
+127.0.0.1       localhost
+::1     localhost ip6-localhost ip6-loopback
+fe00::0 ip6-localnet
+ff00::0 ip6-mcastprefix
+ff02::1 ip6-allnodes
+ff02::2 ip6-allrouters
+172.17.0.2      container1
+172.17.0.2      container1.bridge
+172.17.0.3      container2
+172.17.0.3      container2.bridge
+/ #
+
+```
+
+The default `docker0` bridge network supports the use of port mapping and  `docker run --link` to allow communications between containers in the `docker0` network. These techniques are cumbersome to set up and prone to error. While they are still available to you as techniques, it is better to avoid them and define your own networks instead.
+
+## User-defined networks
+
+Users can create their own user-defined networks that better isolate containers.  Docker provides some default **network drivers** for use defining these networks.  You can create a new **bridge network** or **overlay network**. You can also create a **network plugin** or **remote network**  written to your own specifications.
+
+The next few sections describe each of these drivers in greater detail.
+
+### A bridge network
+
+The easiest user-defined network to create is a `bridge` network. This network is similar to the historical, default `docker0` network. There are some added features and some old features that aren't available.
+
+```
+$ docker network create -d bridge isolated_nw
+8b05faa32aeb43215f67678084a9c51afbdffe64cd91e3f5bb8267475f8bf1a7
+
+$ docker network inspect isolated_nw
+{
+    "name": "isolated_nw",
+    "id": "8b05faa32aeb43215f67678084a9c51afbdffe64cd91e3f5bb8267475f8bf1a7",
+    "driver": "bridge",
+    "containers": {}
+}
+
+$ docker network ls
+NETWORK ID          NAME                DRIVER
+9f904ee27bf5        none                null
+cf03ee007fb4        host                host
+7fca4eb8c647        bridge              bridge
+8b05faa32aeb        isolated_nw         bridge
+
+```
+
+After you create the network, you can launch containers on it using  the `docker run --net=<NETWORK>` option.
```

Comment:
Is it "launch a container on a network"? Or "launch containers, and connect them to the network"?